include/emmc.h

```diff
@@ -39,9 +49,7 @@
  *
  * @param device eMMC /dev path (/dev/mmcblkX)
  * @param bootpart_active will contain the active boot partition
- * (0 for mmcblkXboot0, 1 for mmcblkXboot1, 6 for mmcblkX user partition and
- * -1 for no active boot partition)
- * for user partition)
+ *        (0 for partition 1, 1 for partition 2, -1 for no active boot partition)
```

Comment:
Why not keep boot0/1? "partition 1/2" is less clear in the context of UDA and GP partitions.